index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="assets/css/style.css">
+   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"
+    integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700&display=swap" rel="stylesheet">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+</head>
+
+<!--Body-->
+<body>
+
+  <!--nav-->
+
+  <header>
+    <nav>
+      <h1>my logo</h1>
```

Comment:
- [ ] Please use the `a` tag instead of the `h1` tag. `hx` tags are meant for the main content **headings** or **subheadings** not for the logo, and it is semantically not correct which can result in *bad user experience*. You can take an example of a person and just assume that your website is bigger and have several pages and a user just lost somewhere and that person wants to go back home then in that case apart from `Home` navigation this `logo` with the help of `a` tag makes that person work easier to securely come back on `home-page`.